calblueprint/unloop
# Paperwork/CaseNote does not refresh page on creation/edit. Made CaseNoteCard and ParticipantCard Functional.

This PR does two things.

This PR makes it so that creating/editing paperworks and casenotes does not refresh the page. Instead, a callback is invoked to append the newly created/edited casenote to the component list so that it is rendered.

This PR also made the CaseNoteCard and ParticipantCard functional components. The CaseNoteCard was made functional because state use was basic in the component and making the callback for editing a particular casenote staled the `title` and `description` state for that casenote. Using `componentDidUpdate` or `getDerivedStateFromProps` to unstale the state would've cause additional renders and/or been overkill and complicated, so the component was made functional instead. ParticipantCard was made functional for a similar reason.

## To Test ##
- Simulate casenote/paperwork workflows to make sure that edit/creation works properly.
- If you created the component that I changed, make sure that i didn't accidentally break the component

CC: @didvi
